src/passes/functionPruner/index.ts

```diff
@@ -3,13 +3,7 @@ import { ASTMapper } from '../../ast/mapper';
 import { CallGraphBuilder } from './callGraph';
 import { FunctionRemover } from './functionRemover';
 
-export class UnreachableFunctionPruner extends ASTMapper {
-  // Function to add passes that should have been run before this pass
-  addInitialPassPrerequisites(): void {
```

Comment:
Why was this removed?

Reply:
Oh I didn't realize it got deleted. It must have been when I had to fix some conflicts. I'll add it

Reply:
Might be a good opportunity to add some documentation to this pass as well.